.goreleaser.yml

```diff
@@ -50,3 +50,21 @@ brews:
         type: optional
     test: |
       system "#{bin}/flux --version"
+publishers:
+  - name: aur-pkg-bin
+    ids:
+      - nix
+    cmd: |
+      eval $(ssh-agent -s)
+      ssh-add <(echo "{{ .Env.AUR_SSH_PRIVATE_KEY }}")
```

Comment:
Needs environment variable set.

Reply:
Secret is available in this repository as `AUR_BOT_SSH_PRIVATE_KEY`, configuration for GitHub actions can be found at: https://docs.github.com/en/free-pro-team@latest/actions/reference/encrypted-secrets#using-encrypted-secrets-in-a-workflow